android/app/build.gradle

```diff
@@ -190,7 +193,7 @@ android {
             def abi = output.getFilter(OutputFile.ABI)
```

Comment:
There's a comment in the diff that explain the versionCode numbering, I think you should also report it.

Reply:
I don't get it

Reply:
The diff: https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.64.0 
The comment in it on that file:   `// Example: versionCode 1 will generate 1001 for armeabi-v7a, 1002 for x86, etc. `

Reply:
Isn't the versionCodes.get(abi) already choosing the version code depending on architecture ? 
I don't get what would be needed to changed.

The only thing that changed in the diff is the multiplication factor

Reply:
Read carefully my sentence: 'There's a comment in the diff that explain the versionCode numbering, I think you should also report it.'